client/javascript/scrabble.js

```diff
@@ -66,11 +66,21 @@ Bag.prototype.contains = function(element) {
     return this.contents.indexOf(element) != -1;
 };
 
+var boardLen = 0;
+var middle = Math.floor(boardLen/2);
 function MakeBoardArray()
 {
-    var retval = new Array(15);
-    for (var x = 0; x < 15; x++) {
-        retval[x] = new Array(15);
+	if (boardLen==0) {
+    		console.log('MakeBoardArray size=0');
+		throw "boardLen=0";
+	}
+	else
+	{
+    		console.log('MakeBoardArray size=' + boardLen.toString());
+	}
```

Comment:
This looks like debugging code, can it go?

Reply:
done